deploy/config.secret.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: quay-enterprise-config-secret
+data:
+  config.yaml: |
+    IyBVbmNvbW1lbnQgdG8gdXNlIGEgcmVhbCBteXNxbCBkYiBydW5uaW5nIGluIGRvY2tlcgojIERC
```

Comment:
remove the existing base64 encoded certs and configuration files. This data is specific to any deployed environment.